SplitFiles/control.c

```diff
@@ -175,7 +172,7 @@ void execute_command(ControlBuffer *ctrlBuff, GameParameters* params) {
                 case 'W':
                 if (fetch_bread(e)) {
                     // white bread is driest and quickest to toast
-                    new_slice->old_moisture = 64 + rand()%48;
+                    new_slice->moisture = 64 + rand()%48;
```

Comment:
I think this should be old_moisture for graphing/charting purposes - moisture is then copied from it. Can be the other way around of course

Reply:
ah, you have done it the other way round I see. No problem then

Reply:
Yeah that was the object model change - old_moisture is something the slot monitor needs to know about, but the bread shouldn't care - so I moved it over.  Might be me being an OO wanker, but this was grating on my conceptual model!!